pelican/utils.py

```diff
@@ -613,7 +613,14 @@ def escape_html(text, quote=True):
     return escape(text, quote=quote)
 
 
-def process_translations(content_list, order_by=None):
+def _get_warning_with_source_paths(msg, items, *extra):
+    args = [len(items)]
+    args.extend(extra)
+    args.extend((x.source_path for x in items))
+    return '{}: {}'.format(msg, '\n{}' * len(items)).format(*args)
```

Comment:
Nope! We are using `'%s'` warnings for a reason: Giving logger the ability to do special stuff if needed.

Reply:
Ok, will change this back